src/lib.rs

```diff
@@ -75,7 +75,7 @@ pub trait Field:
     const ONE: Self;
 
     /// Returns an element chosen uniformly at random using a user-provided RNG.
-    fn random(rng: impl RngCore) -> Self;
+    fn random<R: RngCore + ?Sized>(rng: &mut R) -> Self;
```

Comment:
`ff-derive` needs to be updated to generate the new API, and the tests need to be updated to match.